lib/config/decrypt.ts

```diff
@@ -209,3 +221,29 @@ Refer to migration documents here: https://docs.renovatebot.com/mend-hosted/migr
   logger.trace({ config: decryptedConfig }, 'decryptedConfig');
   return decryptedConfig;
 }
+
+function getAzureCollection(): string | undefined {
+  const platform = GlobalConfig.get('platform');
+  if (platform !== 'azure') {
+    return undefined;
+  }
+
+  const endpoint = GlobalConfig.get('endpoint');
+  const endpointUrl = parseUrl(endpoint);
+  if (endpointUrl === null) {
+    // should not happen
+    logger.warn({ endpoint }, 'Unable to parse endpoint for token decryption');
+    return undefined;
+  }
+
+  const azureCollection = endpointUrl.pathname
+    ?.split('/')
+    .filter(Boolean)
```

Comment:
```suggestion
    .filter(is.string)
```
do we really need this?

Reply:
it results in failing tests, the Boolean mainly filters out empty strings

Reply:
ok, then use no empty string 😁

Reply:
done